mlrun/config.py

```diff
@@ -186,7 +186,15 @@
         "dirpath": expanduser("~/.mlrun/db"),
         "dsn": "sqlite:///db/mlrun.db?check_same_thread=false",
         "old_dsn": "",
-        "debug": False,
+        "debug": {
+            "enabled": False,
+            # when enabled, pydevd-pycharm will be started trying to connect to the host & port specified
+            "pydevd_pycharm": {
+                "enabled": False,
+                "port": 4000,
+                "host": "localhost",
+            },
+        },
```

Comment:
```suggestion
        "debug": {
            "mode": "disabled",
            # when enabled, pydevd-pycharm will be started trying to connect to the host & port specified
            "pydevd_pycharm": {
                "mode": "disabled",
                "port": 4000,
                "host": "localhost",
            },
        },
```
I think that will allow us more room to grow if in the future we may want to add another behaviors to the debug modes.